### PR TITLE
Bump Frida, Capstone versions 

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: libafl_frida/Cargo.toml [hunk 1]
```diff
@@ -28,10 +28,10 @@ hashbrown = "0.12"
 libloading = "0.7"
 rangemap = "1.0"
 frida-gum-sys = { version = "0.3", features = [ "auto-download", "event-sink", "invocation-listener"] }
-frida-gum = { version = "0.6.5", features = [ "auto-download", "event-sink", "invocation-listener"] }
+frida-gum = { version = "0.7.1", features = [ "auto-download", "event-sink", "invocation-listener"] }
 regex = "1"
 dynasmrt = "1.2"
-capstone = "0.10"
+capstone = "0.11.0"
```
Comment (domenukk) [inline libafl_frida/Cargo.toml:34]: Why 0.11.0 and not 0.11?

Comment (tokatoka) [reply libafl_frida/Cargo.toml:34]: https://crates.io/crates/capstone says 0.11.0, so I thought it is better

Comment (domenukk) [reply libafl_frida/Cargo.toml:34]: But usually the last digit shouldn't change the api, just patch bugs

Comment (tokatoka) [reply libafl_frida/Cargo.toml:34]: > Then do it at both places

aah, sorry my mistake

File: libafl/src/bolts/shmem.rs [hunk 2]
```diff
@@ -1397,22 +1397,22 @@ impl<T: ShMem> std::io::Seek for ShMemCursor<T> {
...
-                assert!(i64::try_from(current_pos).is_ok());
+                assert!(i64::try_from(current_pos).unwrap());
                 let signed_pos = current_pos as i64;
                 let effective = signed_pos.checked_add(offset).unwrap();
                 assert!(effective >= 0);
                 effective.try_into().unwrap()
             }
         };
-        assert!(usize::try_from(effective_new_pos).is_ok());
+        assert!(usize::try_from(effective_new_pos).unwrap());
```
Comment (domenukk) [inline libafl/src/bolts/shmem.rs:1415]: Unwrapping doesn't need an assert

Comment (domenukk) [reply libafl/src/bolts/shmem.rs:1415]: (right?)

Comment (tokatoka) [reply libafl/src/bolts/shmem.rs]: clippy suggested this silly thing! I'll just add allow()s

**Comment by domenukk** [comment]
No it seems correct, I'm just saying you can remove the assert here

**Pull request merged**